integration/presto/performance-report-of-presto-with-carbon.md

```diff
@@ -0,0 +1,27 @@
+<!--
```

Comment:
**Please remove this section right now as the report is not fair.**

reasons:
1. Spark-carbon TPCH results mentioned in website is not matching with this spark-carbon results. Few queries have huge difference due to machine problem [these machines are not in same rack]
2. Also comparison report should have machine details [RAM, VM/Bare metal] . This was not mentioned.

Reply:
done